protobufs/livekit_models.proto

```diff
@@ -21,7 +21,16 @@ option ruby_package = "LiveKit::Proto";
 
 import "google/protobuf/timestamp.proto";
 
+message KVMetadata {
+  // KV uses Key type from the corresponding model.
+  // Integer keys allow discovering key names from the protobuf itself.
+  map<int32,string> kv = 1;
```

Comment:
What would be the easiest way for a customer to define their own keys? Or is this not a design goal?

Reply:
Good question! It's not a goal, since they already have an opaque `metadata` field that they control. This one is solely for our own use.

Reply:
As discussed on the call, we will now allow user-defined keys too and thus keys will be strings. I updated the PR to reflect that.